### PR TITLE
Add LLVM compatibility to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by manishwalia15**

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
r? llvm

My sense is that this is probably not going to be kept up to date, which makes it not very useful. I'm also not sure we have any real policy on this -- my sense is that the versions are really just "whatever is tested in our CI"?

**Comment by workingjubilee** [comment]
@manishwalia15 Please do not open a new PR for every revision.

**Comment by slanterns** [comment]
https://github.com/rust-lang/rust/pull/131619#discussion_r1797753613

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,3 +40,13 @@ refer to [this section][contributing-bug-reports] and [open an issue][issue temp
 [internals]: https://internals.rust-lang.org
 [rust-discord]: http://discord.gg/rust-lang
 [rust-zulip]: https://rust-lang.zulipchat.com
+
+## External LLVM Version Compatibility
+
+To build `rustc`, it is important to know which versions of LLVM are supported. As of Rust 1.82, the following external LLVM versions are supported:
+
+- **Minimum Supported Version**: LLVM 17
```
Comment (workingjubilee) [inline CONTRIBUTING.md:48]: This is still not the right version, we don't support LLVM 17 anymore. Where are you getting this information?

**Comment by manishwalia15** [comment]
Kindly tell where is the mistake

File: CONTRIBUTING.md (hunk 1 above)
Comment (manishwalia15) [reply CONTRIBUTING.md:48]: image

Comment (workingjubilee) [reply CONTRIBUTING.md:48]: Link?

Comment (workingjubilee) [reply CONTRIBUTING.md:48]: Where is this?

Comment (manishwalia15) [reply CONTRIBUTING.md:48]: [v](https://www.tublian.com/project/724712/131588)

Comment (manishwalia15) [reply CONTRIBUTING.md:48]: https://www.tublian.com/project/724712/131588

Comment (workingjubilee) [reply CONTRIBUTING.md:48]: https://github.com/rust-lang/rust/issues/131588 ?

Comment (manishwalia15) [reply CONTRIBUTING.md:48]: mybe

**Comment by workingjubilee** [comment]
See: https://github.com/rust-lang/rust/pull/130487

The minimum was raised to LLVM 18.

**Comment by manishwalia15** [comment]
shall i update to 18..rest is fine ?

**Comment by slanterns** [comment]
Maybe we should figure out if it's indeed helpful to add this to `CONTRIBUTING.md`.

**Comment by manishwalia15** [comment]
Please do tell

**Comment by nikic** [comment]
https://github.com/rust-lang/rust/blob/master/INSTALL.md#dependencies would be a more appropriate place.

**Comment by manishwalia15** [comment]
@nikic https://github.com/rust-lang/rust/pull/131627
i hv done kindly see

**Comment by slanterns** [comment]
You can update your PR without opening a new one each time.

**Comment by manishwalia15** [comment]
sorry @slanterns i didn knew
next time i wont
please allow for this time

**Pull request closed without merging**